frontend/src/js/store/storehooks.ts

```diff
@@ -182,22 +182,23 @@ export const useAppInit = userId => {
     if (isServiceProvider) {
       return Promise.resolve(dispatch(getRoles()));
     }
+    const selectedAttributes = columnSelection.map(column => ({ attribute: column.key, scope: column.scope }));
     return Promise.all([
       dispatch(getDeviceAttributes()),
       dispatch(getDeploymentsByStatus({ status: DEPLOYMENT_STATES.finished, shouldSelect: false })),
       dispatch(getDeploymentsByStatus({ status: DEPLOYMENT_STATES.inprogress })),
-      dispatch(getDevicesByStatus({ status: DEVICE_STATES.accepted })),
-      dispatch(getDevicesByStatus({ status: DEVICE_STATES.pending })),
-      dispatch(getDevicesByStatus({ status: DEVICE_STATES.preauth })),
-      dispatch(getDevicesByStatus({ status: DEVICE_STATES.rejected })),
+      dispatch(getDevicesByStatus({ status: DEVICE_STATES.accepted, selectedAttributes })),
+      dispatch(getDevicesByStatus({ status: DEVICE_STATES.pending, selectedAttributes })),
+      dispatch(getDevicesByStatus({ status: DEVICE_STATES.preauth, selectedAttributes })),
+      dispatch(getDevicesByStatus({ status: DEVICE_STATES.rejected, selectedAttributes })),
```

Comment:
These requests will not have a big influence on the device list, as the `setDeviceListState` calls in the devices page should take the selection into account (+ except for 'accepted' and maybe 'rejected' devices, the other devices usually won't have any non-standard attribute data).